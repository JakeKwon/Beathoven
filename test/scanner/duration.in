duration dottedQuarter = 1/4 + 1/8;
