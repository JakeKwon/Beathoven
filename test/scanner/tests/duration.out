DURATION
ID
ASSIGN
LIT_INT
DIVIDE
LIT_INT
PLUS
LIT_INT
DIVIDE
LIT_INT
SEP
